content/src/view.js

```diff
@@ -2077,6 +2077,12 @@ function setPaneEditorData(pane, doc, filename, useblocks) {
   var um = editor.getSession().getUndoManager();
   setPrimaryFocus();
 
+  window.addEventListener("storage", function(e) {
+    /* block mode needs to be set to change editor value */
+    setPaneEditorBlockMode(pane, true);
+    dropletEditor.setValue_raw(localStorage.getItem('dropletEditorValue'));
+  }, false);
```

Comment:
Neat that it works.  Here is a list of some things to address:
* Only sync with changes on other tabs that are at exactly the same program (same user, same filename).
* Sync across modes.  There's dropletEditor for block mode and just editor for text mode.  Changing text in one more should sync up with blocks in the other mode.... maybe?  Or maybe only if it actually parses?  That's something to think about.
* Maintain "dirty" bits etc correctly.  So if text changes, the "save" button gets shown at the right time.
* Consider syncing on every change, not only when the line-count changes.
* If you want to reduce syncing, consider using a timer to limit the frequency of changes sent to localStorage.